etc/picongpu/taurus-tud/knl_picongpu.profile.example

```diff
@@ -13,21 +13,37 @@ export MY_NAME="$(whoami) <$MY_MAIL>"
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
-# General modules #############################################################
+# Modules #############################################################
 #
 module purge
```

Comment:
purging here still a good idea? Works everywhere?

Reply:
It does not hurt and there are no modules loaded before - however it throws as warning that tells you that it can not unload the following modules - followed by an empty list.

To avoid confusion I vote for removing the "old" `module purge` 👴

Reply:
alright, I will get it out

Reply:
alright, I will take it out